tools/retry_file_dump/go.mod

```diff
@@ -5,7 +5,7 @@
 
 module github.com/DataDog/datadog-agent/tools/retry_file_dump
 
-go 1.22.0
```

Comment:
Was this intentional?

Reply:
I don't think so, this was done automatically by `inv tidy`. This might be related to my env, I'll look into it.